uint8 ID
int32 waitingTime
